chapters/automate.Rmd

```diff
@@ -1093,17 +1127,29 @@ help :
 Using `man` and online search,
 explain what every part of this recipe does.
 
-## Configuration {#automate-ex-configuration}
+### Configuration {#automate-ex-configuration}
```

Comment:
Why a level-3 header?

Reply:
Because it's an exercise, which are all level 3

Reply:
Um. Yes. I knew that - I was just testing you.